packages/smarthr-ui/src/components/SmartHRAILogo/SmartHRAILogo.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { type ComponentPropsWithoutRef, memo, useMemo } from 'react'
+
+type Props = {
+  /** コンポーネントのタイトル */
+  alt?: string
+  /** コンポーネントの幅 */
+  width?: number | string
+  /** コンポーネントの高さ */
+  height?: number | string
+}
+type ElementProps = Omit<ComponentPropsWithoutRef<'svg'>, keyof Props>
+
+export const SmartHRAILogo = memo<Props & ElementProps>(
+  ({ alt = 'SmartHR（スマートHR） AI', width, height, fill, className, ...rest }) => {
```

Comment:
rest で渡るし特に拡張しないので不要そう。

```suggestion
  ({ alt = 'SmartHR（スマートHR） AI', width, height, fill, ...rest }) => {
```